src/pagination/pagination-model.class.ts

```diff
@@ -2,15 +2,15 @@ export class PaginationModel {
 	/**
 	 * Tracks the current page.
 	 */
-	currentPage: number;
+	currentPage = 0;
```

Comment:
Shouldn't current page be 1?

Reply:
🤔 we may zero index pages ... let me check ...